cmd/aries-agent-lambda/serverless.yml

```diff
@@ -0,0 +1,389 @@
+service: aries-agent-lambda
+
+frameworkVersion: 2.57.0
+variablesResolutionMode: '20210326'
+configValidationMode: off
+disabledDeprecations:
+  - '*'
+
+plugins:
+  - serverless-dotenv-plugin
+  - serverless-dynamodb-local
+  - serverless-plugin-resource-tagging
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: go1.x
+  lambdaHashingVersion: '20201221'
+  stage: ${opt:stage, "local"}
+  logRetentionInDays: 1
+  apiGateway:
+    restApiId: ${env:AWS_REST_API_ID, "0000000000"}
+    restApiRootResourceId: ${env:AWS_REST_API_ROOT_RESOURCE_ID, "0000000000"}
+    websocketApiId: ${env:AWS_WEBSOCKET_API_ID, "0000000000"}
+    websocketsApiRouteSelectionExpression: $request.body.action
+  tracing:
+    lambda: true
+  environment:
+    SERVICE_NAME: "aries-agent-lambda"
+    SERVICE_SLUG: "aries"
+    STAGE: ${opt:stage, "local"}
+  stackTags:
+    ServiceName: ${self:service}
+    ServiceSlug: "aries"
+    ServiceStage: ${opt:stage, "local"}
+
+package:
+  patterns:
+    - '!./**'
+    - ./bin/**
+
+functions:
+  lambda-handler:
+    handler: bin/aries-agent-lambda
+    timeout: 30
+    events:
+      - http:
+          path: /
+          method: any
+      - http:
+          path: /
+          method: options
+      - http:
+          path: /{proxy+}
+          method: any
+      - http:
+          path: /{proxy+}
+          method: options
+      - websocket:
+          route: $connect
+      - websocket:
+          route: $disconnect
+      - websocket:
+          route: $default
+          routeResponseSelectionExpression: $default
+
+resources:
+  Resources:
+    Secret:
+      Type: AWS::SecretsManager::Secret
+      Properties:
+        Name: lambda/${opt:stage, "local"}/${self:service}
+        SecretString: '{}'
+
+    CoordinateMediation:
```

Comment:
These create the needed DyanamoDB tables. Rather than allowing the app higher privileges, we thought this as the better solution. It does mean that adding new tables has to be accompanied with changes here.